aws-aurora-mysql/main.tf

```diff
@@ -16,6 +16,7 @@ module "aurora" {
   db_deletion_protection              = var.db_deletion_protection
   rds_cluster_parameters              = var.rds_cluster_parameters
   iam_database_authentication_enabled = var.iam_database_authentication_enabled
+  iam_roles                           = var.iam_roles
```

Comment:
is this a variable in the aws-aurora module? maybe I missed it, but I don't see one.